heartbeat/aws.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+#
+#   AWS Helper Scripts
+#
+#
+
+: ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
+. ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+
+# Defaults
```

Comment:
Good thinking. You just need to remove _default lines for curl_retries/curl_sleep from the agent as well.

Reply:
Thanks @oalbrigt - I believe keeping this variable in the agent is still valuable, as it’s used in other metadata calls. However, if you feel it's better to remove it, please let me know, and I’m happy to make the change.

![image](https://github.com/user-attachments/assets/d6ad94e2-be33-45cc-b889-c3ffd4423323)

Reply:
I agree, but it should only be in one place. So either in the agents or in `aws.sh`.

Reply:
That makes sense. Thanks for the feedback!